controllers/cloudinit/scripts/00-disable-default-cni.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash -xe
+
+microk8s kubectl delete -f /var/snap/microk8s/current/args/cni-network/cni.yaml
+mv /var/snap/microk8s/current/args/cni-network/cni.yaml /var/snap/microk8s/current/args/cni-network/calico.yaml.old
```

Comment:
maybe keep the names similar?

```suggestion
mv /var/snap/microk8s/current/args/cni-network/cni.yaml /var/snap/microk8s/current/args/cni-network/cni.yaml.old
```

Reply:
Maybe there's a good reason it gets named to calico rather than cni.  if so, ignore.